integration-tests/src/test/groovy/com/okta/sdk/tests/it/UsersIT.groovy

```diff
@@ -250,7 +259,67 @@ class UsersIT extends ITSupport implements CrudTestSupport {
         // 2. Change the user's password
         UserCredentials credentials = user.changePassword(client.instantiate(ChangePasswordRequest)
             .setOldPassword(client.instantiate(PasswordCredential).setValue('Abcd1234'.toCharArray()))
-            .setNewPassword(client.instantiate(PasswordCredential).setValue('1234Abcd'.toCharArray())))
+            .setNewPassword(client.instantiate(PasswordCredential).setValue('1234Abcd'.toCharArray())), true)
+        assertThat credentials.getProvider().getType(), equalTo(AuthenticationProviderType.OKTA)
+
+        // 3. make the test recording happy, and call a get on the user
+        // TODO: fix har file
```

Comment:
Is this TODO still valid?

Reply:
yes, but... i'll start another thread about killing this, i haven't been using those har files.